index.bs

```diff
@@ -313,25 +314,34 @@ partial interface NeuralNetworkContext {
         - *dilations*: an optional sequence of {{long}} of length 2. The dilation factor
             for each spatial dimension of *input*, [dilation_height, dilation_width]. 
             If not present, the values are assumed to be [1,1].
+        - *groups*: an optional {{long}}. The number of groups that input
+            channels and output channels are divided into, default to 1.
         - *layout*: an optional {{OperandLayout}} with value as "nchw" or
             "nhwc". The default value is "nchw". This argument specifies the
             layout format of the input, output and filter tensor.
             Specifically,
 
             "nchw":
                 - input tensor: [batches, input_channels, height, width]
-                - filter tensor: [output_channels, input_channels, height,
-                    width]
+                - filter tensor: [output_channels, input_channels/groups,
+                    height, width]
                 - output tensor: [batches, output_channels, height, width]
             "nhwc":
                 - input tensor: [batches, height, width, input_channels]
-                - filter tensor: [height, width, input_channels,
+                - filter tensor: [height, width, input_channels/groups,
                     output_channels]
                 - output tensor: [batches, height, width, output_channels]
 
     **Returns:** an {{Operand}}. The output 4-D tensor that contains the
     result of the convolution. The logical shape is interpreted according to the
     value of *layout*.
+
+    <div class="note">
+    A *depthwise* conv2d operation is a variant of grouped conv2d where the
+    *groups* = input_channels = output_channels and the shape of filter tensor
+    is [groups, 1, height, width] for "nchw" layout or [height, width, 1,
+    groups] for "nhwc" layout.
+    </div>
```

Comment:
Nit. "A *depthwise* conv2d operation is a variant of grouped conv2d, **used in models like the MobileNet**, where the..."

Reply:
Good idea. I added that into a new commit. Please take another look.